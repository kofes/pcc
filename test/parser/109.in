procedure foo(v : array of char);
begin
end;

procedure foo(v : array of boolean);
begin
end;

begin
end;
