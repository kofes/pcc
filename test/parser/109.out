<function table>
   <procedure><name>FOO</name>
      <params>
         <param>V:p:ARRAY OF BOOLEAN</param>
      </params>
      <body>
      </body>
   </procedure>
   <procedure><name>FOO</name>
      <params>
         <param>V:p:ARRAY OF CHAR</param>
      </params>
      <body>
      </body>
   </procedure>
</function table>
<type table>
   <type>BOOLEAN</type>
   <type>CHAR</type>
   <type>INTEGER</type>
   <type>REAL</type>
</type table>
<var table>
   <var>PI:C:REAL:3.1415926535897932385</var>
</var table>
<block statement>
</block statement>
